desc/geometry/surface.py

```diff
@@ -116,14 +124,24 @@ def __init__(
                 sym = False
 
         self._R_basis = DoubleFourierSeries(
-            M=self._M, N=self._N, NFP=NFP, sym="cos" if sym else False
+            M=MR,
+            N=NR,
+            NFP=NFP,
+            NFP_umbilic_factor=NFP_umbilic_factor,
+            sym="cos" if sym else False,
         )
         self._Z_basis = DoubleFourierSeries(
```

Comment:
same as above

Reply:
I don't remember making that change. Reverting back...